test-requirements.txt

```diff
@@ -0,0 +1,6 @@
+coverage>=3.7.1
```

Comment:
Uh only tox is a dependency for this project. I did a pip freeze and didn't check the contents.